.github/workflows/dev_builds.yaml

```diff
@@ -70,3 +82,8 @@ jobs:
       - name: Build and push bundle image to Open Source ECR
         run: make bundle-build bundle-push \
           BUNDLE_IMG=${OPENSOURCE_ECR_URL}/${BUNDLE_IMAGE_NAME}:${{ steps.extract_tag.outputs.tag }}
+
+      - name: Build and push catalog image to Open Source ECR
+        run: make catalog-build catalog-push \
+          CATALOG_IMG=${OPENSOURCE_ECR_URL}/${CATALOG_IMAGE_NAME}:${{ steps.extract_tag.outputs.tag }} \
+          BUNDLE_IMGS=${OPENSOURCE_ECR_URL}/${BUNDLE_IMAGE_NAME}:${{ steps.extract_tag.outputs.tag }}
```

Comment:
Don't you need to log in with the catalog credentials before pushing this?

Reply:
Logging is made earlier: https://github.com/SumoLogic/sumologic-kubernetes-collection-helm-operator/pull/69/files#diff-b8f02dfa7af10ede6fd8bbf61a9eb9f4ca0c9fb4ae6e6a72f7fa03d502b92212R43

Reply:
Yeah, but you log in using `_BUNDLE` credentials, whereas [here](https://github.com/Sanyaku/opensource-infrastructure/pull/42) you've provisioned `_CATALOG` ones. Is this ok?

Reply:
Only `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables are needed to get ECR credentials using `aws ecr-public get-login-password --region us-east-1` [here](https://github.com/SumoLogic/sumologic-kubernetes-collection-helm-operator/blob/50e04fdf8c760e8190df306d7ee0adc7ffbbd7d9/Makefile#L65)
Only one logging to ECR is needed to push images for operator, bundle and catalog.

Reply:
I added additional step to log in to Open Source ECR to push catalog image, for public ECR only one loggin is enough.